modules/ROOT/pages/wireguard-config.adoc

```diff
@@ -0,0 +1,259 @@
+:experimental:
+= Configuring FCOS to use WireGuard
+
+== Introduction
+
+https://www.wireguard.com/[WireGuard] is a novel VPN that runs inside the Linux Kernel and uses state-of-the-art cryptography (the "Noise" protocol). It aims to be faster, simpler, leaner, and more useful than IPSec, while avoiding the massive headache. It intends to be considerably more performant than OpenVPN. WireGuard is designed as a general purpose VPN for running on embedded interfaces and super computers alike, fit for many different circumstances. It runs over UDP.
+
+- https://www.wireguard.com/[Conceptual Overview]
+- https://www.wireguard.com/quickstart/[Quickstart]
+- https://www.wireguard.com/papers/wireguard.pdf[Whitepaper]
+
+This guide is going to demonstrate how to setup a single connection between a FCOS server and one client computer. It goes over the basic client configuration, but it does not cover installing WireGuard on your clients.
```

Comment:
I think this could be rephrased to be a bit more clear.  For example, it would be useful to say that WireGuard is already included on FCOS, so no installation is necessary there.  

Maybe frame the rest of the guide where the FCOS system is acting a peer for a WG connection; users will need to setup a second FCOS system (or other system) to act as the other peer for the WG connection.

Reply:
This guide is more geared for setting up a WireGuard VPN with FCOS as the server and you can configure your clients (laptop, phone, etc) to connect to it. 

I will add in a line about FCOS having WireGuard support out of the box.